client/state/reader/site-blocks/reducer.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
FFTI: have you looked into [createReducer](https://github.com/Automattic/wp-calypso/blob/master/client/state/utils.js#L167)? I saw it for the first time recently and thought that it helps with the boilerplate :)